ethereum/hardhat.config.ts

```diff
@@ -133,7 +158,7 @@ const config: HardhatUserConfig = {
 
 interface RigsTablesConfig {
   tablelandPrivateKey: string | undefined;
-  tablelandAlchemyKey?: string;
```

Comment:
I think this change breaks the `deployRigs` and `deployRigsAndPilots` scripts since they still refer to the alchemy key

Reply:
pushed a fix